v2/tests/log_context_property_basic_types_ut/log_context_property_basic_types_ut.c

```diff
@@ -309,37 +309,1626 @@ static void int32_t_free_returns(void)
 /* LOG_CONTEXT_PROPERTY_TYPE_IMPL(int64_t).get_type */
 
 /* Tests_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_010: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(int64_t).get_type shall returns the property type LOG_CONTEXT_PROPERTY_TYPE_int64_t. ]*/
-static void int32_t_get_type_returns_LOG_CONTEXT_PROPERTY_TYPE_int64_t(void)
+static void int64_t_get_type_returns_LOG_CONTEXT_PROPERTY_TYPE_int64_t(void)
 {
     // arrange
     setup_mocks();
 
     // act
-    LOG_CONTEXT_PROPERTY_TYPE result = LOG_CONTEXT_PROPERTY_TYPE_IMPL(int32_t).get_type();
+    LOG_CONTEXT_PROPERTY_TYPE result = LOG_CONTEXT_PROPERTY_TYPE_IMPL(int64_t).get_type();
 
     // assert
-    POOR_MANS_ASSERT(result == LOG_CONTEXT_PROPERTY_TYPE_int32_t);
+    POOR_MANS_ASSERT(result == LOG_CONTEXT_PROPERTY_TYPE_int64_t);
     POOR_MANS_ASSERT(actual_and_expected_match);
 }
 
-/* very "poor man's" way of testing, as no test harness and mocking framework are available */
-int main(void)
+/* LOG_CONTEXT_PROPERTY_TYPE_IMPL(uint64_t_t).to_string */
```

Comment:
>uint64_t_t [](http://example.com/codeflow?start=34&length=10)

Is this some copy&pasta'd&replace'd type?

Reply:
An obvious case of here and everywhere

Reply:
Yep, case of search/replace